### ISSUE TITLE
Missing string in json file?

### ISSUE THREAD

**Issue by luce-carevic** [rumoroso/headingsmap_translations#4]
I was using HeadingsMap on a website with iframe and I think you have a missing string in your json file (for me it's not related to the bug you've fixed).

<img width="911" alt="" src="https://github.com/rumoroso/headingsmap_translations/assets/6073336/c85da1bd-065f-4372-9fad-0bb49e5a6d16">

>select the document to obtain its heading/sectioning structure

**Comment by rumoroso** [comment]
Hi, very good catch. I didn't realize of it. I guess this is a leftover of when all the strings were in the code and only had one language version. Then I went one by one moving them to the locale files and... it seems that I forgot one (well, at least one... who knows if there are more). 
It was very good that you found it. I just removed it from the code and add a new entry in the json file. :)

It was a coincidence that one guy reported one bug and I am fixing it (in fact, I think I just did it), so when the code is done and the translations merged, I'll release the update with everything. 
I would like to ask you something else... I promise it will be the last time. Since the next release will include these changes, I will add something to the release notes (everytime I release, I do it), so besides the translation of that text that was in the code, I would need the French translation of the release notes. I just updated all in the main branch, so the changes are already uploaded in the json file (if I remember correctly, the new string is in the line 113) and also in the last entry (at the beginning of the release notes). I promise it is the last time I ask for your help. You were very kind doing everything (in fact, you are already by far the person that has contributed more).
